src/main/scala/tv/teads/github/api/util/LazyLogging.scala

```diff
@@ -1,10 +1,10 @@
 package tv.teads.github.api.util
 
+import com.typesafe.scalalogging.Logger
 import org.slf4j.LoggerFactory
-import org.slf4j.Logger
 
 trait LazyLogging {
 
-  protected lazy val logger: Logger = LoggerFactory.getLogger(getClass.getName)
+  protected lazy val logger = Logger(LoggerFactory.getLogger(getClass.getName))
```

Comment:
That wasn't why I meant: drop our own `LazyLogging` altogether and use the one from scalalogging.